src/ripple/app/tx/impl/CreateOffer.cpp

```diff
@@ -569,22 +645,18 @@ CreateOffer::applyGuts (ApplyView& view, ApplyView& view_cancel)
     bool const bFillOrKill (uTxFlags & tfFillOrKill);
     bool const bSell (uTxFlags & tfSell);
 
-    STAmount saTakerPays = tx().getFieldAmount (sfTakerPays);
-    STAmount saTakerGets = tx().getFieldAmount (sfTakerGets);
+    auto saTakerPays = tx()[sfTakerPays];
```

Comment:
I think `tx()` is the victim of a weird merge / rebase. It was defined as an accessor to `ctx_.tx`, and seemed to be the standard across the transactors. I would have sworn that it there when I started, but now the diff is showing that I added it, so I can only assume that it got removed somewhere along the way, and one of my rebases silently put it back.

I'll take it out.